srml/staking/src/lib.rs

```diff
@@ -934,6 +963,25 @@ decl_module! {
 			ensure_root(origin)?;
 			<Invulnerables<T>>::put(validators);
 		}
+
+		/// Add reward points to validator.
+		///
+		/// At the end of the era each the total payout will be distributed among validator
+		/// relatively to their points.
+		fn add_reward_points_to_validator(origin, validator: T::AccountId, points: u32) {
+			ensure_root(origin)?;
```

Comment:
I remember we discussed this: who will eventually call this? I assume root is like a temporary placeholder.

Reply:
Ok seeing the event handler from authorship: maybe this can be a simple module function?

Reply:
yes maybe I don't know the implication, what I want is that this function is accessible the most broadly possible. I feel like a simple module function could be enough indeed.

Reply:
the only advantage of root call instead of module function is that sudo can execute it. but this seems useless actually.

OK I'll move to module function